doc/md/guides/orms/hibernate.mdx

```diff
@@ -0,0 +1,439 @@
+  for them, based on the desired state of their schema instead of crafting them by hand.
+
+## Automatic migration planning for Hibernate
+Hibernate is a popular ORM widely used in the Java community. Hibernate allows users to
+manage their database schemas using its [schema generation](https://docs.jboss.org/hibernate/orm/current/userguide/html_single/Hibernate_User_Guide.html#schema-generation) feature,
+which is usually sufficient during development and in many simple cases.
+
+However, at some point, teams need more control and decide to employ
+the [versioned migrations](/concepts/declarative-vs-versioned#versioned-migrations)
+methodology, which is a more robust way to manage your database schema.
+Once this happens, the responsibility for planning migration scripts and making
+sure they are in line with what Hibernate expects at runtime is moved to developers.
+
+Atlas can automatically plan database schema migrations for developers using Hibernate.
+Atlas plans migrations by calculating the diff between the _current_ state of the database,
+and its _desired_ state.
+
+In the context of versioned migrations, the current state can be thought of as the database schema that would have been created by applying
+all previous migration scripts.
+
+The desired schema of your application can be provided to Atlas via an [External Schema Datasource](/atlas-schema/projects#data-source-external_schema)
+which is any program that can output a SQL schema definition to stdout.
+
+To use Atlas with Hibernate, users can utilize the [Hibernate Atlas Provider](https://github.com/ariga/atlas-provider-hibernate),
+a small program that can be used to load the schema of a Hibernate project into Atlas.
+
+In this guide, we will show how Atlas can be used to automatically plan schema migrations for
+Hibernate users.
+
+## Prerequisites
+
+* A local [Hibernate](https://hibernate.org/orm/) project.
+
+If you don't have a Hibernate project handy, checkout [hibernate getting started page](https://docs.jboss.org/hibernate/orm/current/userguide/html_single/Hibernate_User_Guide.html)
+
+## Using the Atlas Hibernate Provider
+
+In this guide, we will use the [Hibernate Atlas Provider](https://github.com/ariga/atlas-provider-hibernate)
+to automatically plan schema migrations for a Hibernate project.
+
+### Installation
+
+Install Atlas from macOS or Linux by running:
+```bash
+curl -sSf https://atlasgo.sh | sh
+```
+See [atlasgo.io](https://atlasgo.io/getting-started#installation) for more installation options.
+
+Install the provider by adding this plugin to your gradle project:
+
+<Tabs>
+<TabItem value="Gradle Kotlin" label="Gradle Kotlin" default>
+
+```kotlin
+plugins {
+    id("io.atlasgo.hibernate-provider") version "0.1"
+}
+```
+</TabItem>
+<TabItem value="Gradle Groovy" label="Gradle Groovy" default>
+
+```
+plugins {
+    id "io.atlasgo.hibernate-provider" version "0.1"
+}
+```
+</TabItem>
+<TabItem value="Maven" label="Maven">
+
+```
+Coming soon!
```

Comment:
maybe drop maven from the docs, until we will support it

Reply:
I prefer keeping it, letting users know it's coming and shows our intention (and maybe even create some engagement if they ask us about it)

I think it's up to @hilakashai to decide?

Reply:
Hmm, don't have a strong opinion on this.  I think it's nice to show the intention. Looks good either way to me